src/main.ts

```diff
@@ -36,8 +36,9 @@ export class Main extends Command {
   }
 
   protected _help() {
-    const HHelp: typeof Help = require('@oclif/plugin-help').default
-    const help = new HHelp(this.config)
+    const helpPlugin = this._helpPlugin()
+    const HHelp = require(helpPlugin).default
+    const help: HelpBase = new HHelp(this.config)
```

Comment:
If we have to do this command.ts and main.ts, any reason not to put it in the `this._helpPlugin()` function? I don't think you need access to `helpPlugin` or `HHelp` so just having that method return the `help` seems fine.

Reply:
Honestly, I like renaming `getHelpPluginPackage` to just `getHelpPlugin` and doing this logic in there. If this was in the try/catch, then if `new HHelp` blows up because there wasn't a default or it wasn't a class, it wouldn't fail with a cryptic oclif error.